cmsl1t/playground/metfilters.py

```diff
@@ -0,0 +1,18 @@
+import cmsl1t.geometry as geo
+
+
+def pfMetFilter(event):
+
+    reject_if = [
+        not event._metFilterReco.MetFilters.hbheNoiseFilter,
+        not event._metFilterReco.MetFilters.hbheNoiseIsoFilter,
+        not event._metFilterReco.MetFilters.globalSuperTightHalo2016Filter,
+        not event._metFilterReco.MetFilters.ecalDeadCellTPFilter,
+        not event._metFilterReco.MetFilters.goodVerticesFilter,
+        not event._metFilterReco.MetFilters.eeBadScFilter,
+        not event._metFilterReco.MetFilters.badPFMuonFilter,
+        not event._metFilterReco.MetFilters.badChCandFilter
+    ]
+    if any(reject_if):
```

Comment:
probably worth changing to `accept_if` and `if all(accept_if)` since all entries are `not`-ed